.bit/responses/2.1-Week 2 Step 1.md

```diff
@@ -15,10 +15,10 @@ Because of amazing APIs, you don't need to be an expert in machine learning and
 ### ✅ Task:
 
 **Create a request in Postman to send an image of a person to the Azure Face API to return the subject's emotions**
-- [ ] Create a Face [API Endpoint](https://smartbear.com/learn/performance-monitoring/api-endpoints/#:~:text=For%20APIs%2C%20an%20endpoint%20can,to%20carry%20out%20their%20function.&text=The%20place%20that%20APIs%20send,lives%2C%20is%20called%20an%20endpoint.)
```

Comment:
You should add numbers next to these tasks like Shreya did. Also add in header links so students can just click on the link to jump to the section they need help on